From a628be884d79ba9e0611be9236b51eff2cafa61b Mon Sep 17 00:00:00 2001
From: "[Anjali MA]" <[Anjali.MA_ext@Ltts.com]>
Date: Thu, 12 Feb 2026 11:51:05 +0530
Subject: [PATCH] here is the complete details

---
 patches.txt | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 patches.txt

diff --git a/patches.txt b/patches.txt
new file mode 100644
index 0000000..9a090b7
--- /dev/null
+++ b/patches.txt
@@ -0,0 +1,2 @@
+welcome to ltts
+
-- 
2.34.1

